rhsm/rhsm-context.c

```diff
@@ -369,6 +369,10 @@ rhsm_context_new (void)
       entitlement_cert_dir = rhsm_key_file_get_interpolated_string (conf, "rhsm", "entitlementCertDir", NULL);
     }
 
+  /* Check rhsm.conf exists */
+  if (!g_file_test (conf_file, G_FILE_TEST_EXISTS))
+    g_warning ("Not found config file %s", conf_file);
```

Comment:
If this is an expected/supported case, then let's not warn about it.  We could just use `g_debug()`.

Reply:
SGTM. For rhel-coreos base image, we do not have `subscription-manager`, so we do not have `rhsm.conf`.